core/arch/arm/plat-ls/ls_i2c_test.c

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2020 NXP
+ * Test suite for I2C Controller
+ */
+
+#include <drivers/ls_i2c.h>
+#include <initcall.h>
+#include <kernel/delay.h>
+#include <kernel/tee_time.h>
+#include <mm/core_memprot.h>
+#include <string.h>
+#include <stddef.h>
+
+#define PCF2129_CTRL3_BIT_BLF  BIT(2) /* Battery Low Flag*/
+#define PCF2129_SLAVE_ADDRESS  0x51
+
+struct __packed pcf2129_regs {
```

Comment:
The preferred location of `__packed` is after the closing `}` for a struct.

Reply:
ok, will update.